app/views/thredded/messageboards/_form.html.erb

```diff
@@ -29,3 +29,8 @@
       </li>
     </ul>
 <% end %>
+
+<%= button_to "Delete", messageboard_path(messageboard), :method => 'delete',
```

Comment:
Please add a `Thredded.show_messageboard_delete_button` setting.
For many use cases, having the ability to delete whole boards from the UI is undesirable.

Reply:
Done! I hope I've done this right. Defaults to false, added tests for both cases.